.github/workflows/runner-publish.yml

```diff
@@ -0,0 +1,75 @@
+# This workflows will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+name: Upload Python Package
+
+on:
+  repository_dispatch:
+    types: publish-runners
+
+jobs:
+  ssh_deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build distribution
+      run: |
+        cd runners/mlcommons_box_ssh
+        python setup.py sdist bdist_wheel
+        cp -avr dist ../../
+    - name: Publish
+      uses: pypa/gh-action-pypi-publish@master
+      with:
+        password: ${{ secrets.MLCOMMONS_SSH_PYPI_TOKEN }}
```

Comment:
If we're using separate secrets and jobs, does it make sense to have separate workflows instead?

Reply:
The tokens are for PYPI projects. Separate workflows don't solve it.